pkg/controller/volume/events/event.go

```diff
@@ -30,5 +30,6 @@ const (
 	ProvisioningCleanupFailed = "ProvisioningCleanupFailed"
 	ProvisioningSucceeded     = "ProvisioningSucceeded"
 	WaitForFirstConsumer      = "WaitForFirstConsumer"
```

Comment:
Can this be removed?

Reply:
Nm I see it's still used. Can we also update this event message to be more clear that "first consumer" == pod?

Reply:
yes, it's still used to indicate that no pod is created to consume it. I suggest not changing it.
`WaitForFirstConsumer` is also used in storage class volume binding mode. When people see it in the event message, I think it's clear to them.